src/providers/krb5/krb5_utils.c

```diff
@@ -570,6 +573,18 @@ parse_krb5_map_user(TALLOC_CTX *mem_ctx, const char *krb5_map_user,
         }
     }
 
+    /* conversion names to fully-qualified names */
+    for (int i = 0; i < size; i++) {
+        fq_name = sss_create_internal_fqname(tmp_ctx,
+                                             name_to_primary[i].id_name,
+                                             dom_name);
+        name_to_primary[i].id_name = talloc_strdup(name_to_primary, fq_name);
```

Comment:
I added conditions on sss_create_internal_fqname() calls.

I use strdup for two reason:
* is const, of course,
* without it the CI test fails -- it is because of need more room for FQ name.